.github/workflows/ci.yml

```diff
@@ -50,9 +50,20 @@ jobs:
         php_version: ${{ matrix.php_version }}
         php_extensions: gettext intl xsl pcov
 
+    - name: Install Node
+      uses: actions/setup-node@v3
+      with:
+        node-version: 16.x
+
+    - name: Install JavaScript dependencies
```

Comment:
I think this step could be avoided by making use of [npm package support](https://github.com/actions/setup-node/blob/main/README.md#caching-global-packages-data) in the setup-node action above. That also includes caching for npm dependencies.

Basically, just adding `cache: npm` to the setup-node action should ensure that npm dependencies are installed.

Reply:
Good point. We agreed to make this improvement later.